index.html

```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
+              you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="about-footer">
+            <div class="about-btn">
+              <span><a href="#">Get my resume</a></span>
+            </div>
+            <div class="about-border"></div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <ul class="language-sec">
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec.svg" alt="rec" />
+              <h4>Language</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">Ruby</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec2.svg" alt="rec" />
+              <h4>Frameworks</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">React.js</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSpec</a></li>
+                <li><a href="#">Capybara</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec3.svg" alt="rec" />
+              <h4>skills</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">Database Management</a></li>
+                <li><a href="#">Version Control</a></li>
+                <li><a href="#">CLI</a></li>
+                <li><a href="#">API Design</a></li>
+                <li><a href="#">Web Design</a></li>
+              </ul>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <div class="contact">
+          <div class="contact-input">
+            <input type="text" placeholder="Full Name" name="name" required />
+          </div>
+          <div class="contact-input">
+            <input
+              type="email"
+              placeholder="Email Address"
+              name="email"
+              required
+            />
+          </div>
+          <div class="info">
+            <textarea placeholder="Write something ..." rows="6"></textarea>
+          </div>
+        </div>
+        <div class="contact-btn">
+          <div class="story-btn">
+            <span><a href="#">Get in touch</a></span>
+          </div>
```

Comment:
it's better to use button tag as a tag is to redirect user to a new page or bloc in the page but here we have a form and the button is needed to submit data in the form so we can benefit from the type= submit property of the button so when clicked all data in inputs inside the form will be submited to the destination in the action of our form

Reply:
done :heavy_check_mark: